nautobot_ssot/integrations/servicenow/jobs.py

```diff
@@ -65,6 +65,12 @@ def config_information(cls):
         }
 
     def load_source_adapter(self):
+        """Load Nautobot adapter."""
+        self.logger.info("Loading current data from Nautobot...")
+        self.source_adapter = NautobotDiffSync(job=self, sync=self.sync, site_filter=self.site_filter)
+        self.source_adapter.load()
```

Comment:
Sorry if I'm missing something here, but did we not load the source adapter before or why is thos method new?

Reply:
Alright